src/Bang.Analyzers/Bang.Analyzers.csproj

```diff
@@ -1,12 +1,44 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
+        <!-- Project configuration. -->
         <TargetFramework>netstandard2.0</TargetFramework>
         <LangVersion>11</LangVersion>
-        <ImplicitUsings>enable</ImplicitUsings>
         <Nullable>enable</Nullable>
-        <LangVersion>latest</LangVersion>
+        <ImplicitUsings>enable</ImplicitUsings>
         <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+        
+        <!-- NuGet package configuration. -->
+        <RootNamespace>Bang</RootNamespace> <!-- This is necessary because of the way things are packed.-->
+        <AssemblyName>Bang.Analyzers</AssemblyName>
+        <NoPackageAnalysis>true</NoPackageAnalysis>
+        <IncludeBuildOutput>false</IncludeBuildOutput>
+        <DevelopmentDependency>true</DevelopmentDependency>
+        <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+        <Description>Source code analyzers for Bang</Description>
+
+        <PackageId>Murder.Bang.Analyzers</PackageId>
+        <PackageVersion>0.0.1-alpha</PackageVersion>
+        <Authors>Murder Authors</Authors>
+        <Company>Murder Engine</Company>
+        <PackageLicenseFile>LICENSE</PackageLicenseFile>
+
+        <!-- Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        
     </PropertyGroup>
+    <ItemGroup>
+        <None Include="..\..\LICENSE" Pack="true" PackagePath="" Visible="false"/>
+
+        <!-- Include .dll in the correct folder. -->
+        <None Include="$(OutputPath)/$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+
+        <!-- Files that keep track of what analyzer are/aren't included in releases. -->
+        <None Remove="AnalyzerReleases.Shipped.md" />
+        <None Remove="AnalyzerReleases.Unshipped.md" />
+<!--        <AdditionalFiles Include="AnalyzerReleases.Shipped.md" />-->
```

Comment:
will we need to add `AdditionalFiles` at some point?

Reply:
Oh, we do right now 😂